app/src/main/res/values/strings.xml

```diff
@@ -46,4 +46,8 @@
     <string name="contest_status_bar_title">Contest Status</string>
     <string name="status_waiting_submissions_title">Scores sent!</string>
     <string name="status_waiting_submissions_description">We\'ll notify you soon.\nWe\'re waiting on %1$s…</string>
+
+    <string name="status_results_available_contest_title">%1$s</string>
```

Comment:
Do we need this string resource? looks like we're just using it as set text variable.